rtl/eclrtl/rtlnewkey.cpp

```diff
@@ -1632,6 +1649,16 @@ void RowFilter::addFilter(const IFieldFilter & filter)
         numFieldsRequired = fieldNum+1;
 }
 
+void RowFilter::addFilter(const RtlRecord & record, const char * filterText)
+{
+    //assertex(filter.queryField() == filters.ordinality()); //MORE - fill with wild filters and replace existing wild
```

Comment:
Is this comment still accurate? Is there a plan to address it? A Jira? Should it be fixed in this PR?

Reply:
I think the comment can be deleted - it was related to when they are used for keys, and that needs more work anyway.